File: library/src/main/java/cn/cqray/android/dialog/amin/ZoomOut.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 缩放退出
 * @author Cqray
 */
public class ZoomOut extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.zoomOut();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/ZoomOut.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 缩放退出
 * @author Cqray
 */
class ZoomOut : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.zoomOut()
    }
}